src/main/kotlin/no/nav/syfo/simba/MapInntektsmelding.kt

```diff
@@ -93,7 +91,7 @@ fun mapInntektsmelding(
                 ?.endringer
                 ?.map(RefusjonEndring::tilEndringIRefusjon)
                 .orEmpty(),
-        førsteFraværsdag = bestemmendeFravaersdag,
+        førsteFraværsdag = null,
```

Comment:
Via altinn-ruta settes førstefraværsdag fortsatt, men regner med at det er greit..
Ser at inntektsmelding.forstefraværsdag blir initialisert i en del tester til LocalDate.now(), kanskje det gir mening å sette det feltet til null i disse testene også..? 🤔  Evt i noen av dem, for å huske på at fra nå av er det alltid null fra simba, men kan fortsatt ha verdi når det kommer fra altinn / LPS..  Tror ikke det vil ha noe å si for det vi tester, evt kan man kanskje bare legge til en kommentar her - om at førsteFraværsdag er null her, men ikke nødvendigvis null fra Altinn-løpet. Men vet ikke.. Bare vanskelig å ha oversikt over alle krumspring og forskjeller.

Reply:
Ingen endringer på Altinn-ruta.

Hvilke tester tenkte du på? Jeg ser de som er knyttet opp mot Altinn-ruta, men er det relevant å endre på dem?

Det blir litt styr når vi skal trykke to ulike løp sammen, men det blir forhåpentligvis bedre på nyåret, når Spleis får til en egen håndtering for IM-er fra nav.no.

Reply:
Nei, ikke sikkert at de testene er så relevante, de er vel enten knyttet til altinn-ruta eller bare frittstående. Men bare evt en kommentar i MapInntektsmelding om at den settes til null fordi spleis ikke lengre leser dette feltet for ny im, så slipper jeg å måtte sjekke git-logg når jeg kommer tilbake fra ferie og lurer på hvorfor den er null :)

Reply:
Will do!